konvoy: Update docs for konvoy release v1.5.0-beta.8 (#3020)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5.0-beta/gpu/index.md

1.  NVIDIA GPU with Fermie architecture version 2.1 or greater.

> 1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.

Reviewer: ```suggestion
1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.
```

> 1.  On Centos 7, we recommend using the latest kernel version in production, while the default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, please see the following sections for detailed configuration. Please note that Ubuntu does not have the kernel version restriction.

Reviewer 2: ```suggestion
1.  On Centos 7, we recommend using the latest kernel version in production. The default AMI's kernel version `3.10.0-1062` should work as well. If you want to use a dedicated kernel version on Centos, refer to the following sections for detailed configuration. Note that Ubuntu does not have the kernel version restriction.
```

      enabled: true
```

### Nvidia Driver Configuration Based on Kernel Version

> Additional configuration may be needed if your kernel version of the GPU nodes is not identical to the latest one from the yum repository. In this case, you need to pick the right image tag for your Nvidia Driver container. For example, the following list is the supported kernel versions from the Nvidia upstream `nvidia/driver` image:

Reviewer 2: ```suggestion
Additional configuration may be needed if your kernel version of the GPU nodes is not identical to the latest one from the yum repository. You need to pick the right image tag for your Nvidia Driver container. For example, the following list is the supported kernel versions from the Nvidia upstream `nvidia/driver` image:
```

```bash
440.64.00-1.0.0-3.10.0-1127.8.2.el7.x86_64-centos7
440.64.00-1.0.0-3.10.0-1127.el7.x86_64-centos7
440.64.00-1.0.0-3.10.0-1062.18.1.el7.x86_64-centos7
440.33.01-3.10.0-1062.12.1.el7.x86_64-centos7
440.33.01-3.10.0-1062.9.1.el7.x86_64-centos7
440.33.01-3.10.0-1062.7.1.el7.x86_64-centos7
440.33.01-3.10.0-1062.4.3.el7.x86_64-centos7
418.87.01-3.10.0-1062.4.3.el7.x86_64-centos7
418.87.01-3.10.0-1062.4.1.el7.x86_64-centos7
418.87.01-3.10.0-1062.1.2.el7.x86_64-centos7
418.40.04-3.10.0-957.21.3.el7.x86_64-centos7
418.40.04-3.10.0-957.21.2.el7.x86_64-centos7
418.40.04-3.10.0-957.12.2.el7.x86_64-centos7
418.40.04-3.10.0-957.12.1.el7.x86_64-centos7
418.40.04-3.10.0-957.10.1.el7.x86_64-centos7
396.37-3.10.0-957.5.1.el7.x86_64-centos7
396.37-3.10.0-957.1.3.el7.x86_64-centos7
396.37-3.10.0-862.14.4.el7.x86_64-centos7
396.37-3.10.0-862.14.4-centos7
396.37-3.10.0-862.11.6-centos7
396.37-3.10.0-862.9.1-centos7
```

> This list is from [Nvidia Public Hub Repository][nvidia_public_hub_repository]. You would need to identify your kernel version first. For example,

Reviewer 2: ```suggestion
This list is from [Nvidia Public Hub Repository][nvidia_public_hub_repository]. You need to identify your kernel version first. For example,
```

```bash
[centos@ip-10-0-128-77 ~]$ uname -r
3.10.0-1062.12.1.el7.x86_64
```

Then, find the corresponding pre-built `nvidia/driver` image from the list above. In this case, `440.33.01-3.10.0-1062.12.1.el7.x86_64-centos7` is the right image tag to pick. Configure the `cluster.yaml` to adopt this image tag for your nvidia driver container:

```yaml
    - name: nvidia
      enabled: true
      values: |
        nvidia-driver:
          image:
            tag: "440.33.01-3.10.0-1062.12.1.el7.x86_64-centos7"
```

### GPU on Air-gapped On-prem Cluster

> Please follow the [Konvoy Air-gapped Installations](../install/install-airgapped/) doc. Re-tag the `nvidia/driver` image with the corresponding tag that is identified from the above section, and push it to your local registry. For example:

Reviewer 2: ```suggestion
Follow the [Konvoy Air-gapped Installations](../install/install-airgapped/) doc. Re-tag the `nvidia/driver` image with the corresponding tag, identified from the above section, and push it to your local registry. For example:
```
